cgi-bin/LJ/Console/Command/ScreenList.pm

```diff
@@ -0,0 +1,67 @@
+# This code was forked from the LiveJournal project owned and operated
+# by Live Journal, Inc. The code has been modified and expanded by
+# Dreamwidth Studios, LLC. These files were originally licensed under
+# the terms of the license supplied by Live Journal, Inc, which can
+# currently be found at:
+#
+# http://code.livejournal.org/trac/livejournal/browser/trunk/LICENSE-LiveJournal.txt
+#
+# In accordance with the original license, this code and all its
+# modifications are provided under the GNU General Public License.
+# A copy of that license can be found in the LICENSE file included as
+# part of this distribution.
```

Comment:
I had this discussion with @kaberett recently, and I'm not sure this is adequately documented: new Console commands need to go in DW/Console/Command instead of LJ/Console/Command, with a Dreamwidth copyright notice instead of a LiveJournal one.  You can look at the existing commands in DW/Console/Command to see what that copyright should look like, and do let me know if you have any questions about the details.